examples/aws/aws-cdk-examples/openid-connect-github/resources.js

```diff
@@ -30,18 +32,27 @@ exports.createResources = () => [
             },
             Action: "sts:AssumeRoleWithWebIdentity",
             Condition: {
-              StringEquals: {
+              StringLike: {
                 [`${getId({
                   type: "OpenIDConnectProvider",
                   group: "IAM",
                   name: "oidp::token.actions.githubusercontent.com",
                   path: "live.Url",
-                })}:aud`]: "sts.amazonaws.com",
+                })}:sub`]: [
```

Comment:
You need both audience and a sub claim. See here: https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services#configuring-the-role-and-trust-policy

Reply:
it is also missing from https://github.com/dannysteenman/aws-cdk-examples/blob/ff2459c818dcf9994584edb7b98445e98d030ed8/openid-connect-github/main.ts#L43-L47, isn't ?

Reply:
Oh, yeah, I see that the `clientIds` parameter contain the audience.